src/GraphQL.ApiTests/ApiApprovalTests.PublicApi.GraphQL.NewtonsoftJson.approved.txt

```diff
@@ -0,0 +1,51 @@
+namespace GraphQL.NewtonsoftJson
```

Comment:
Is GraphQL.Json.Newtonsoft a better namespace? Then we can have a other like GraphQL.Json.SystemText. I think this is more clear, that the can use for the same purpose.

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/pull/1448#discussion_r354559306

Reply:
Thanks, overlooked it.